packages/components/src/menu-group/index.js

```diff
@@ -33,9 +37,14 @@ export function MenuGroup( {
 	return (
 		<div className={ classNames }>
 			{ label &&
-				<div className="components-menu-group__label" id={ labelId }>{ label }</div>
+				<div
+					className={ `components-menu-group__label ${ ( isScreenReaderLabel ) ? 'screen-reader-text' : '' }` }
```

Comment:
Here if isScreenReaderLabel is false we add an unnecessary space. What if we use the classnames util e.g:
```
className={ classnames( 'components-menu-group__label', {
['screen-reader-text']: isScreenReaderLabel
} }
```